docs/InstallGuide.md

```diff
@@ -58,15 +46,14 @@ Modify the following section in /etc/openvnet/vnmgr.conf on the vnmgr node.
       addr {
         protocol "tcp"
         host "192.168.100.2"
+        public ""
         port 9102
       }
     }
 
-- **host** : The IP address or host name of the ZeroMQ node.
-- **port** : The port number for connecting with ZeroMQ. If the vnmgr process is run on the same node as vna or webapi, specify a different value from vna and webapi.
-
+- **host** : The private IP address of the vnmgr node.
+- **port** : The port number for the ZeroMQ connection. If the vnmgr process is run on the same node as vna o
```

Comment:
This is the first time ZeroMQ and vnmgr node are mentioned. The reader of this guide doesn't know what those are. Before installing everything there should be a brief explanation of the 3 OpenVNet processes and that they communicate through ZeroMQ.

Also, the "If the vnmgr process is run on the same node as vna o" sentence could be better like this:

Each ZeroMQ process needs a TCP port to listen on. `9102` is the default port but since we're currently installing multiple processes (webapi, vnmgr and vna) in the same machine, change the vnmgr port to `9103`.

(Also change the port in the above config example to 9103 to reflect this)